pkg/core/state/migrations/postgres/4_v0.24.0.down.sql

```diff
@@ -0,0 +1,5 @@
+DROP FUNCTION IF EXISTS "cloudquery"."calculate_policy_executions_stats" CASCADE;
```

Comment:
if it's going to be included in 24.0 then merge it with 3_, otherwise filename should be `4_v0.24.1`